specification/flag-evaluation.md

```diff
@@ -93,19 +93,6 @@ client.getMetadata().getName(); // "my-client"
 
 ##### Requirement 1.3.1
 
-> The `client` **MUST** provide methods for flag evaluation, with parameters `flag key` (string, required), `default value` (boolean | number | string | structure, required), `evaluation context` (optional), and `evaluation options` (optional), which returns the flag value.
-
-```typescript
-// example flag evaluation
-var myValue = client.getValue("my-flag", false);
-```
-
-##### Condition 1.3.2
-
-> The language type system differentiates between strings, numbers, booleans and structures.
-
-###### Conditional Requirement 1.3.2.1
-
```

Comment:
I think we should still include the parameters list, we can just merge it with the "typed flag evaluation" requirement.

Reply:
I have merged it.